internal/cli/runner_inspect.go

```diff
@@ -0,0 +1,136 @@
+package cli
+
+import (
+	"fmt"
+	"github.com/dustin/go-humanize"
+	"github.com/golang/protobuf/ptypes"
+	"reflect"
+	"strings"
+
+	"github.com/golang/protobuf/jsonpb"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	pb "github.com/hashicorp/waypoint/pkg/server/gen"
+	"github.com/posener/complete"
+)
+
+type RunnerInspectCommand struct {
+	*baseCommand
+
+	flagJson bool
+}
+
+func (c *RunnerInspectCommand) Run(args []string) int {
+	// Initialize. If we fail, we just exit since Init handles the UI.
+	if err := c.Init(
+		WithArgs(args),
+		WithFlags(c.Flags()),
+		WithNoLocalServer(), // no auth in local mode
+		WithNoConfig(),
+	); err != nil {
+		return 1
+	}
+
+	if len(c.args) == 0 {
+		c.ui.Output("Runner name required.", terminal.WithErrorStyle())
+		return 1
+	}
+	id := c.args[0]
+
+	resp, err := c.project.Client().GetRunner(c.Ctx, &pb.GetRunnerRequest{RunnerId: id})
+	if err != nil && status.Code(err) != codes.NotFound {
+		c.ui.Output(clierrors.Humanize(err), terminal.WithErrorStyle())
+		return 1
+	}
+
+	if c.flagJson {
+		var m jsonpb.Marshaler
+		m.Indent = "\t"
+		str, err := m.MarshalToString(resp)
```

Comment:
I think we should build up some other things in a map of sorts first and then render them to json, similar to what we do with [deployments](https://github.com/hashicorp/waypoint/blob/b49074ec5c8e86a1ce37dc9e7358b22f71b6f050/internal/cli/deployment_list.go#L150-L152). As is, this leaves some of the information behind that you normally get without using `-json`. 

Examples:

```
[waypoint][runner-inspect](4)$ waypoint runner inspect 01FSW91WAMHBQZ542NKKR2ST1Q

» Runner:
              ID: 01FSW91WAMHBQZ542NKKR2ST1Q
  Adoption State: new
            Kind: remote
          Labels: map[]
          Online: false

[waypoint][runner-inspect](4)$ waypoint runner inspect -json 01FSW91WAMHBQZ542NKKR2ST1Q
{
        "id": "01FSW91WAMHBQZ542NKKR2ST1Q",
        "remote": {

        }
}
```

Reply:
Heh, interestingly enough, I believe most of the `inspect` commands format the raw proto response as json directly, so this would match the other inspect commands.

Reply:
yeah, I vote leave it and make it a to-do to redo item the other inspects with good parsing